### DISCUSSION TITLE
How is the initial Git build actioned during SDK installation?

### DISCUSSION THREAD

**Discussion by PhilipOakley** [git-for-windows/git#4376] [Q&A] [answered]
Installing the SDK should bootstrap the downloading of the SDK, and the clone and initial compilation/[build  of the Git-for-Windows code from source](https://github.com/git-for-windows/build-extra/blob/main/README.md#L5). 

I'm trying to document that process and find myself stuck in trying to find the particular lines of code that detect the initial sdk install and then effectively run `sdk build git`. 

It's not clear if it's hidden in the `profile`, or in the `sdk`, or some where else? Or maybe the README is slightly out of date and the user is expected to run the appropriate `sdk` command?

Any pointers?

**Comment by dscho** [comment, marked as answer]
> I'm trying to document that process and find myself stuck in trying to find the particular lines of code that detect the initial sdk install and then effectively run `sdk build git`.

I am guessing that this is a question instead of a statement?

If so, I assume that you are asking "where is the code in the SDK installer that builds Git right after starting the Git SDK Bash for the first time?". And the answer is: it was removed in git-for-windows/build-extra@d1a55d3c568b23bb711f0669130035b3469dad4e

**Reply by PhilipOakley** [reply]
Thanks @dscho for that clarification. 
I do remember browsing that change but I probably had it in my head that the build step had some how been moved somewhere, rather than it simply being removed.

At least that clarifies that the SDK no longer does a build step. 

I'll get a PR to update the build-extra README.md to make that point, though I'm away for a couple of weeks...
